contrib/syntax-checking/config.el

```diff
@@ -15,6 +15,12 @@
 (defvar syntax-checking-enable-tooltips t
   "If non nil some feedback are displayed in tooltips.")
 
+(defvar syntax-checking-ispell-program "aspell"
+  "External program to use for ispell. Defaults to \"aspell\".")
+
+(defvar syntax-checking-ispell-dictionary "english"
+  "Dictionary to use for ispell. Defaults to \"english\".")
+
 ;; Command Prefixes
```

Comment:
Why even bother setting these -- just curious -- since you check and set it to aspell and english respectively...just take the value of this var :)